.circleci/test-deploy.yml

```diff
@@ -92,7 +99,9 @@ jobs:
           command: mvn package
           working_directory: .circleci/simple-maven-package-test/maven-package-test
       - cloudsmith-maven/upload_package_using_cli:
+          release_type: "stable"
```

Comment:
Should we have another test job that checks that the release_type "unstable" works?

i.e
_test-upload_unstable_package_using_cli:_

Reply:
- This would be a good idea in theory. However, because you can only upload one maven package, it's not possible to test a snapshot and a release at the same time.
- I did however run the CircleCI build against "stable" and "unstable" release types, and can confirm that releases and snapshots are being uploaded to the correct repositories and all the tests passed. You can see the results [here](https://app.circleci.com/pipelines/github/ft-circleci-orbs/cloudsmith-maven-orb).